packages/aws/src/lib/CostAndUsageReports.ts

```diff
@@ -677,3 +674,37 @@ export default class CostAndUsageReports {
     }
   }
 }
+
+export const tagNameToAthenaColumn = (tagName: string): string => {
+  let columnName = 'resource_tags_'
+
+  for (const char of tagName) {
+    if (char == ':') {
+      columnName = columnName + '_'
+    } else {
+      if (isUpperCase(char) && !lastCharacterIs(columnName, '_')) {
+        columnName = columnName + '_'
+      }
+
+      columnName = columnName + char.toLowerCase()
+    }
+  }
+
+  return columnName
+}
+
+const isUpperCase = (text: string): boolean => {
```

Comment:
These functions could potentially live in a common / helpers directory rather than the CUR file

Reply:
True - if there's a need for them elsewhere, we could move them then.